examples/preprocessing/plot_all_scaling.py

```diff
@@ -0,0 +1,328 @@
+
+make_plot(1)
+
+##########################################################################
+# MinMaxScaler
+# ------------
+#
+# ``MinMaxScaler`` rescales the data set such that all feature values are in
+# the range [0, 1] as shown in the right panel below. However, this scaling
+# compress all inliers in the narrow range [0, 0.005] for the transformed
+# number of households.
+#
+# As ``StandardScaler``, ``MinMaxScaler`` is very sensitive to the presence of
+# outliers.
+
+make_plot(2)
+
+#############################################################################
+# MaxAbsScaler
+# ------------
+#
+# ``MaxAbsScaler`` differs from the previous scaler such that the absolute
+# values are mapped in the range [0, 1]. On positive only data, this scaler
+# behaves similarly to ``MinMaxScaler`` and therefore also suffers from the
+# presence of large outliers.
+
+make_plot(3)
+
+##############################################################################
+# RobustScaler
+# ------------
+#
+# Unlike the previous scalers, the centering and scaling statistics of this
+# scaler are based on percentiles and are therefore not influenced by a few
+# number of very large marginal outliers. Consequently, the resulting range of
+# the transformed feature values is larger than for the previous scalers and,
+# more importantly, are approximately similar: for both features most of the
+# transformed values lie in a [-2, 3] range as seen in the zoomed-in figure.
+# Note that the outliers themselves are still present in the transformed data.
+# If a separate outlier clipping is desirable, a non-linear transformation is
+# required (see below).
+
+make_plot(4)
+
+###################################################################
+# QuantileTransformer (uniform output)
+# ------------------------------------
+#
+# ``QuantileTransformer`` applies a non-linear transformation such that the
+# probability density function of each feature will be mapped to a uniform
+# distribution. In this case, all the data will be mapped in the range [0, 1],
+# even the outliers which cannot be distinguished anymore from the inliers.
+#
+# As ``RobustScaler``, ``QuantileTransformer`` is robust to outliers in the
+# sense that adding or removing outliers in the training set will yield
+# approximately the same transformation on held out data. But contrary to
+# ``RobustScaler``, ``QuantileTransformer`` will also automatically collapse
+# any outlier by setting them to the a priori defined range boundaries (0 and
+# 1).
+
+make_plot(5)
+
+##############################################################################
+# QuantileTransformer (Gaussian output)
+# -------------------------------------
+#
+# ``QuantileTransformer`` has an additional ``output_distribution`` parameter
+# allowing to match a Gaussian distribution instead of a uniform distribution.
+# Note that this non-parametetric transformer introduces saturation artifacts
+# for extreme values.
+
+make_plot(6)
```

Comment:
Here is the rendered example:

https://10116-843222-gh.circle-artifacts.com/0/home/ubuntu/scikit-learn/doc/_build/html/stable/auto_examples/preprocessing/plot_all_scaling.html#results